app/models/katello/glue/elastic_search/content_view_puppet_environment.rb

```diff
@@ -5,26 +5,6 @@ module Glue::ElasticSearch::ContentViewPuppetEnvironment
     def self.included(base)
       base.send :include, Ext::IndexedModel
```

Comment:
you can get rid of this too.  It actually causes an issue when trying to delete/remove a content view version